rock-dkms-bin/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: acxz <akashpatel2008@yahoo.com>
 pkgname=rock-dkms-bin
 pkgver=3.5
-_pkgver=$pkgver-32
+_pkgver=$pkgver-30
 pkgrel=4
```

Comment:
`pkgrel` should be bumped up by 1 whenever package is updated when the `pkgver` is still the same. See: https://wiki.archlinux.org/index.php/PKGBUILD#pkgrel

Reply:
Done. I pushed my forked master, but I am kinda new to the process. I am not sure if I need to do something else.

Reply:
np